spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -73,18 +81,19 @@ def process(options)
 
     # kills travis sometimes so leaving it here as tested documentation
     describe "process termination", if: false do
-      before { subject.demultiplexer.stub(:stop){ throw :got_stopped } }
+      before { allow(subject.demultiplexer).to receive(:stop) { throw :got_stopped } }
 
       context "when process quits" do
         before do
           stub_const("Quiter", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              Process.kill("QUIT", 0)
-            end
-          })
+                               def process(options)
+                                 Process.kill("QUIT", 0)
+                               end
+                             })
```

Comment:
`end` at 92, 29 is not aligned with `Class.new(DispatchRider::Handlers::Base) {` at 88, 31 or `stub_const("Quiter", Class.new(DispatchRider::Handlers::Base) {` at 88, 10